tests/end_to_end_integration/run_test.sh

```diff
@@ -75,13 +75,17 @@ EOF
 registry="$1"
 tag="$2"
 fv3net="$PWD"
-random="$(openssl rand --hex 6)"
-name="integration-test-$random"
-GCS_OUTPUT_URL="gs://vcm-ml-scratch/test-end-to-end-integration/$name"
+# random="$(openssl rand --hex 6)"
```

Comment:
I think we may still want to add some randomness to this string since the tests can be invoked several times in one day on a single commit. The timestamp only includes the day.